integrations/ollama/src/haystack_integrations/components/embedders/ollama/text_embedder.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict, List, Optional
+
+import requests
+from haystack import component
+
+
+@component
+class OllamaTextEmbedder:
+    def __init__(
+        self,
+        model: str = "orca-mini",
+        url: str = "http://localhost:11434/api/embeddings",
+        generation_kwargs: Optional[Dict[str, Any]] = None,
+        timeout: int = 120,
+    ):
+        """
+        :param model: The name of the model to use. The model should be available in the running Ollama instance.
+            Default is "orca-mini".
+        :param url: The URL of the chat endpoint of a running Ollama instance.
+            Default is "http://localhost:11434/api/embeddings".
+        :param generation_kwargs: Optional arguments to pass to the Ollama generation endpoint, such as temperature,
```

Comment:
These `generation_kwargs` look exactly like the ones supported by the generator. Can they really be used for the embedder too?

Reply:
From what I have seen from the ollama docs, yes -> unfortunately they haven't said if they can't

Reply:
I was actually dubious about this, but as @AlistairLR112 says, there are no obvious and documented differences in Ollama between the text generation endpoints and the embedding creation endpoint.

For now, let's keep the parameter as is.